From c36f343f88f65a9765085f1346da6d6591ef86a5 Mon Sep 17 00:00:00 2001
From: Lukas Slebodnik <lslebodn@redhat.com>
Date: Mon, 26 Oct 2015 07:00:50 +0100
Subject: [PATCH 7/7] BUILD: Accept krb5 1.14 for building the PAC plugin

Reviewed-by: Sumit Bose <sbose@redhat.com>
(cherry picked from commit 8fe87c3d35bf301cbb6ed7d441b588327d831924)
(cherry picked from commit 3dd118ee870d4370e8bfff8bd71d7e9954ccac06)
---
 src/external/pac_responder.m4 | 3 ++-
 1 file changed, 2 insertions(+), 1 deletion(-)

diff --git a/src/external/pac_responder.m4 b/src/external/pac_responder.m4
index 2941d63ab5bc9f2c993cabf1080a04f812611751..a9e142dc945e157d87ab8478fa63b261415a7e8d 100644
--- a/src/external/pac_responder.m4
+++ b/src/external/pac_responder.m4
@@ -22,7 +22,8 @@ then
         Kerberos\ 5\ release\ 1.10* | \
         Kerberos\ 5\ release\ 1.11* | \
         Kerberos\ 5\ release\ 1.12* | \
-        Kerberos\ 5\ release\ 1.13*)
+        Kerberos\ 5\ release\ 1.13* | \
+        Kerberos\ 5\ release\ 1.14*)
             krb5_version_ok=yes
             AC_MSG_RESULT([yes])
             ;;
-- 
2.5.0

